widget.css

```diff
@@ -1,44 +1,48 @@
-
-#social-widget {
-    background: #fff;
-    width: 100%;
-}
-
-#social-widget header {
-    background: #dadada;
+.social-widget {
```

Comment:
Could you please explain a bit, why your version is better and what was dreadfully wrong with the original CSS.